src/api/app/models/staging/workflow.rb

```diff
@@ -77,4 +78,11 @@ def update_staging_projects_managers_group
     # the object is reloaded and we lost the changes.
     self.managers_group = new_managers_group
   end
+
+  def add_reviewer_group
+    role = Role.find_by_title('reviewer')
+    return unless role
```

Comment:
[nitpick] if that role does not exist, a lot of other code crashes. You're a little over-defensive here